packages/autocomplete-preset-algolia/package.json

```diff
@@ -34,11 +34,9 @@
     "@algolia/autocomplete-shared": "1.8.3"
   },
   "devDependencies": {
-    "@algolia/client-search": "4.16.0",
     "algoliasearch": "4.16.0"
   },
   "peerDependencies": {
-    "@algolia/client-search": ">= 4.9.1 < 6",
```

Comment:
We don't need this dependency since it has been moved to the `autocomplete-shared` package.

Reply:
you still need to list transitive peer dependencies though

Reply:
Good point, thanks!